packages/preview-middleware-client/src/flp/initConnectors.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Initializes UI5 connectors based on the current UI5 version.
+ *
+ * For UI5 versions below 1.72, this function dynamically requires and executes a FakeLrepConnector.
+ * For UI5 versions 1.72 and above, it defines a local connector that reuses the WorkspaceConnector.
+ * This setup allows for flexibility in using different connectors based on the UI5 version.
+ *
+ * @example
+ * intiConnectors(); // Simply call the function without any arguments.
+ * @returns {void}
+ */
+export function initConnectors(): void {
+    const version = sap.ui.version;
+    const minor = Number(version.split(/[.-]/)[1]);
```

Comment:
Please use `split('.')` and `parseInt` 😉

Reply:
Changed in: [fc3bc56](https://github.com/SAP/open-ux-tools/pull/1597/commits/fc3bc56727fe07650e28b77a19ac47bf087eb8fe)